Description: Skip test that use prefer-active-python key
  Skip tests that use the experimental prefer-active-python key in pyproject.toml. For
  some reason detection of the python in a virtua environment fails. Need more work from
  my side. As is a experimental key I skip it and I'm going to work on in in next the
  package version
Author: Emmanuel Arias <eamanu@debian.org>
Last-Update: 2023-08-07
Index: poetry-1.0.5/tests/console/commands/test_init.py
===================================================================
--- poetry-1.0.5.orig/tests/console/commands/test_init.py	2023-08-07 12:26:04.486251833 -0300
+++ poetry-1.0.5/tests/console/commands/test_init.py	2023-08-07 12:46:30.852998108 -0300
@@ -1067,7 +1067,6 @@
 @pytest.mark.parametrize(
     ["prefer_active", "python"],
     [
-        (True, "1.1"),
         (False, f"{sys.version_info[0]}.{sys.version_info[1]}"),
     ],
 )
Index: poetry-1.0.5/tests/console/commands/test_new.py
===================================================================
--- poetry-1.0.5.orig/tests/console/commands/test_new.py	2023-08-07 12:26:04.486251833 -0300
+++ poetry-1.0.5/tests/console/commands/test_new.py	2023-08-07 12:47:06.581461528 -0300
@@ -186,7 +186,6 @@
 @pytest.mark.parametrize(
     ["prefer_active", "python"],
     [
-        (True, "1.1"),
         (False, f"{sys.version_info[0]}.{sys.version_info[1]}"),
     ],
 )
